src/www/interfaces.php

```diff
@@ -1832,16 +1843,13 @@ function toggle_allcfg() {
                           <th colspan="2"><?=gettext("General configuration"); ?></th>
                         </tr>
                       </thead>
-                      <tbody>
+                      <tbody id="hidecfg">
                         <tr>
-                          <td style="width:22%"><a id="help_for_descr" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Description"); ?></td>
-                          <td style="width:78%">
-                            <input name="descr" type="text" id="descr" value="<?=$pconfig['descr'];?>" />
-                            <div class="hidden" data-for="help_for_descr">
-                              <?= gettext("Enter a description (name) for the interface here."); ?>
-                            </div>
-                          </td>
+                          <td style="width:22%"></td>
+                          <td style="width:78%"><?=gettext("The interface must be enabled to configure detailed settings"); ?></td>
```

Comment:
```suggestion
                          <td style="width:78%"><?=gettext("The interface must be enabled before configuring it further."); ?></td>
```

Reply:
Hesitant (pendantic?) because "before configuring it _further_ " kind of feels like just first time setup, but here it needs to be enabled _every_ time, to even see (not just access) any other config settings. Pedantic I know.  

The suggested edit doesn't quite feel right to me but it's really minor wording and not worth much time. So please pick a wording that you like, and apply it :) I'll be fine with it.

Reply:
"detailed" settings is misleading. we now have two "general configuration" tabs and we unhide the second.

maybe the section headers could be "basic configuration" and "generic configuration".. ."general" seems too, well, general and we already "configure" in the case of the "basic" things like name, lock and operational state.

it would be nice to align the guiding text to the reality of the page descriptions. maybe the suggestion isn't there 100% yet, but that's why we discuss... so what do you think?

Reply:
- Top part (enable/lock/name):  `Basic [Interface] Configuration` or `Basic [Interface] Settings`
- Everything else:  `Network Settings` or `Networking`

Would that work? The distinction seems to be that the top (always visible) part is basic configuration/basic settings, such as on/off and locked/unlocked;  the rest are its related network settings such as DHCP/MTU.